packages/simcore-sdk/src/simcore_sdk/node_ports_v2/nodeports_v2.py

```diff
@@ -148,7 +147,7 @@ async def set_multiple(
 
         raises ValidationError
         """
-        tasks = deque()
+        tasks = []
```

Comment:
```suggestion
        tasks: duque[Coroutine] = duque()
```

Reply:
Please read this (and there are several in google):
https://devpress.csdn.net/python/62f4ec94c6770329307faa09.htm

when appending, a list is on par with a deque. using a deque for no performance gain in a location that does not a priori need it, that gets then converted to a list is not something that makes the code clearer so I prefer not to do it.